db/migrate/20160203152552_create_newsletter_finders.rb

```diff
@@ -0,0 +1,7 @@
+class CreateNewsletterFinders < ActiveRecord::Migration
+  def change
+    create_table :newsletter_finders do |t|
+      t.timestamps null: false
```

Comment:
Is there a reason that there is a table for this model? Could it just be a [PORO](http://blog.jayfields.com/2007/10/ruby-poro.html)?